packages/selinux/build.yaml

```diff
@@ -1,16 +1,12 @@
-requires:
-- name: "base"
-  category: "distro"
-  version: ">=0"
+image: centos:7
 env:
 - GITHUB_ORG={{ ( index .Values.labels "github.owner" ) }}
 - GITHUB_REPO={{ ( index .Values.labels "github.repo" ) }}
 
 {{ if .Values.distribution }}
```

Comment:
guess we can drop this if at all now?

Reply:
ok ok sounds good, maybe we can just leave a small comment here on the reason why? is there a bugzilla entry open for the issue?

Reply:
nope, only our own issue for tracking this.